packages/runtime/container-runtime/src/gc/garbageCollection.ts

```diff
@@ -865,6 +873,78 @@ export class GarbageCollector implements IGarbageCollector {
 		return this.summaryStateTracker.refreshLatestSummary(result);
 	}
 
+	/**
+	 * Process a GC message.
+	 * @param message - The GC message from the container runtime.
+	 * @param local - Whether it was send by this client.
+	 */
+	public processMessage(message: ContainerRuntimeGCMessage, local: boolean) {
+		switch (message.contents.type) {
+			case "Sweep": {
```

Comment:
@markfields I wish with the new typing, it was possible to use `GarbageCollectionMessageType.Sweep` here. Is there a way? Same when sending the message.

Reply:
Good point. Yeah, turns out there is - see my suggestion in gcDefinitions